### ISSUE TITLE
The plugin directory is empty on heroku

### ISSUE THREAD

**Issue by mapreferee** [tigrish/redmine_s3#9]
Hi! I tried to get the git file by using

git clone https://github.com/tigrish/redmine_s3 vendor/plugins/redmine_s3
git add config/s3.yml vendor/plugins/redmine_s3/
git commit -m "Add s3 support"

And

mkdir vendor/plugins/redmine_s3 directory
wget https://github.com/tigrish/redmine_s3/tarball/master
tar -xvzf master 
mv tigrish-redmine_s3-235ee25/\* .
git add .
git commit -m "Add s3 support"

I tried these two ways. but the plugin directory is empty:

$ ls vendor/plugins/redmine_s3/
$

Here is what I got:
https://github.com/mapreferee/redmine_aws3/tree/master/vendor/plugins

Wonder if I have done something wrong here?

**Comment by tigrish** [comment]
You may need to remove vendor/plugins from .gitignore

**Comment by mapreferee** [comment]
No, vendor/plugins is not in the .gitignore. [https://github.com/mapreferee/redmine_aws3/blob/master/.gitignore]

Can this plugin be installed when redmine is running on heroku already?

p.s. pushing code to github/heroku works when I install redmine from the scratch and delete the .git directory in your git file.
